python/create_mask.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime
+import glob
+import sys
+import mahotas
+import numpy as np
+import matplotlib.pyplot as plt
+
+if len(sys.argv) > 1:
+    path = sys.argv[1]
+else:
+    path=''
+
+f=glob.glob(path+"/meta/*ch_location*.txt")
+
+file=open(f[0],"r")
+
+siz=4096
+iarr=np.zeros((siz,siz),dtype=np.byte)
+slate,circ=np.array(iarr),np.array(iarr)
+x,y=[],[]
+fill=0
+i=0
+
+for line in file:
+	if line[0:3] != 'Dat':
+		if line [0:3] == 'Sol':
+			r=line[21:-1]
+		elif line[0:3] == '   ':
+			x,y=np.append(x,int(float(line[6:13]))),np.append(y,int(float(line[20:27])))
+		elif line[0:3] == 'Pix':
+			pix2arc=float(line[22:-1])
```

Comment:
I finding all this reading characters from a file very fragile. Are you sure these character positions will always be the same??

Reply:
Yes, I have made the system that automatically creates the files being read. The structure of these files should alway be the same as these specified values read have a similar structure to a header.